packages/builder/builder/src/plugins/sourceBuild.ts

```diff
@@ -108,7 +108,9 @@ export function builderPluginSourceBuild(
             // webpack.js.org/configuration/module/#ruleresolve
             chain.module
               .rule(useTsLoader ? CHAIN_ID.RULE.TS : CHAIN_ID.RULE.JS)
-              .resolve.mainFields.merge(['...', sourceField]);
+              // source > webpack default mainFiedls.
+              // when source is not exist, other mainFields will effect.
+              .resolve.mainFields.merge([sourceField, '...']);
```

Comment:
Should we update the `conditionNames` orders too?

Reply:
No, `conditionNames` is different from `mainFields`. During condition matching, earlier entries have higher priority and take precedence over later entries  in the `exports` object.

Reply:
get👌